addon/wait-for-promise.ts

```diff
@@ -1,7 +1,7 @@
+import AsyncTestWaiter from './async-test-waiter';
 import { DEBUG } from '@glimmer/env';
-import TestWaiter from './test-waiter';
 
-const PROMISE_WAITER = new TestWaiter<Promise<unknown>>('promise-waiter');
+const PROMISE_WAITER = new AsyncTestWaiter<Promise<unknown>>('promise-waiter');
```

Comment:
We should be using our own public API here, let's swap this to `buildWaiter()`

Reply:
Good call.